models/trigger_map_item.go

```diff
@@ -399,6 +414,9 @@ func (item TriggerMapItemModel) validateNoPullRequestConditionsSet(idx int, fiel
 	if isStringLiteralOrRegexSet(item.PullRequestLabel) {
 		return fmt.Errorf("trigger item #%d: both %s and pull_request_label defined", idx+1, field)
 	}
+	if isStringLiteralOrRegexSet(item.PullRequestComment) {
```

Comment:
Shouldn't `commit_message `and `changed_files` be added too? (Or maybe it's not that simple, because those are now valid for push or PR.)

Reply:
This is not resolved yet: maybe I wasn't clear. it's good that you removed the check from pushes because now it's also valid in PRs, but now there's no check that these don't appear in tag trigger items, is there?

Reply:
gotcha, I think I fixed it